nixos-test/update.sh

```diff
@@ -11,13 +11,16 @@ bin/compose2nix \
   -check_systemd_mounts \
   -include_env_files=true \
   -generate_unused_resources=true \
-  -use_upheld_by
+  -use_upheld_by \
```

Comment:
Please revert. Let's keep the Docker config as-is to ensure that we're testing the non-option path as well in the integration test.